components/email/src/Email.svelte

```diff
@@ -1695,6 +1713,18 @@
                               </button>
                             </div>
                           {/if}
+                          {#if _this.show_forward}
+                            <div class="forward">
+                              <button
+                                title="Forward"
+                                aria-label="Forward"
+                                on:click|stopPropagation={(e) =>
+                                  handleForwardClick(e, msgIndex)}
```

Comment:
Why don't we pass the `message` instead of `msgIndex` here? We're only using the index to look up the message in the handler function.